pkg/test/nodepool.go

```diff
@@ -53,6 +53,7 @@ func NodePool(overrides ...v1beta1.NodePool) *v1beta1.NodePool {
 	}
 	if override.Status.Conditions == nil {
 		override.StatusConditions().SetTrue(v1beta1.ConditionTypeValidationSucceeded)
+		override.StatusConditions().SetTrue(v1beta1.ConditionTypeNodeClassReady)
```

Comment:
Shouldn't this be done automatically by operatorpkg?

Reply:
`operatorpkg` would take care of the base condition which is `Ready` for nodePool. This is `NodeClassReady` condition which is one of the conditions that will affect nodePool's readiness.